scripts/st2bootstrap-deb.sh

```diff
@@ -420,6 +444,7 @@ ok_message() {
 
 trap 'fail' EXIT
 STEP="Setup args" && setup_args $@
+STEP="Check st2 host" && check_st2_host
```

Comment:
IMO "Check st2 host" is too general.
Maybe it worth saying explicitly something about conflicting ports?